soc/riscv/riscv-ite/common/chip_chipregs.h

```diff
@@ -833,7 +833,7 @@
 
 #define GCR2			ECREG(EC_REG_BASE_ADDR + 0x16F1)
 #define CK32OE			BIT(6)
-#define SMB4E			BIT(5)
+#define SMB3E			BIT(5)
```

Comment:
What is happening here? If this is a bug fix, can you please put it in a separate commit (same PR)?

Reply:
Done